tensorflow_lite_support/python/test/task/audio/audio_classifier_test.py

```diff
@@ -87,11 +125,77 @@ def _build_test_data(classifications):
 
 class AudioClassifierTest(parameterized.TestCase, base_test.BaseTestCase):
 
+  def setUp(self):
+    super().setUp()
+    self.test_audio_path = test_util.get_test_data_path(_SPEECH_AUDIO_FILE)
+    self.model_path = test_util.get_test_data_path(_FIXED_INPUT_SIZE_MODEL_FILE)
+
+  def test_create_from_file_succeeds_with_valid_model_path(self):
+    # Creates with default option and valid model file successfully.
+    classifier = _AudioClassifier.create_from_file(self.model_path)
+    self.assertIsInstance(classifier, _AudioClassifier)
+
+  def test_create_from_options_succeeds_with_valid_model_path(self):
+    # Creates with options containing model file successfully.
+    base_options = _BaseOptions(file_name=self.model_path)
+    options = _AudioClassifierOptions(base_options=base_options)
+    classifier = _AudioClassifier.create_from_options(options)
+    self.assertIsInstance(classifier, _AudioClassifier)
+
+  def test_create_from_options_fails_with_invalid_model_path(self):
+    # Invalid empty model path.
+    with self.assertRaisesRegex(
+        ValueError,
+        r"ExternalFile must specify at least one of 'file_content', "
+        r"'file_name' or 'file_descriptor_meta'."):
+      base_options = _BaseOptions(file_name='')
+      options = _AudioClassifierOptions(base_options=base_options)
+      _AudioClassifier.create_from_options(options)
+
+  def test_create_from_options_succeeds_with_valid_model_content(self):
+    # Creates with options containing model content successfully.
+    with open(self.model_path, 'rb') as f:
+      base_options = _BaseOptions(file_content=f.read())
+      options = _AudioClassifierOptions(base_options=base_options)
+      classifier = _AudioClassifier.create_from_options(options)
+      self.assertIsInstance(classifier, _AudioClassifier)
+
+  def test_create_from_options_succeeds_with_valid_model_content(self):
+    # Creates with options containing model content successfully.
+    with open(self.model_path, 'rb') as f:
+      base_options = _BaseOptions(file_content=f.read())
+      options = _AudioClassifierOptions(base_options=base_options)
+      classifier = _AudioClassifier.create_from_options(options)
+      self.assertIsInstance(classifier, _AudioClassifier)
+
+  def test_create_input_tensor_audio_from_classifier_succeeds(self):
```

Comment:
Please add test_create_audio_record_from_embedder_succeeds and test_create_input_tensor_audio_from_embedder_succeeds to the audio_embedder_test.py as well.

Reply:
@khanhlvg I've added it to the AudioEmbedder as well.